build/context.go

```diff
@@ -50,9 +50,11 @@ func (c *CloudBuilder) sendBuildContext() (err error) {
 	}()
 	// send to remote server
 	workdir := fmt.Sprintf(common.DefaultWorkDir, c.local.Cluster.Name)
-	if err = c.SSH.Copy(c.RemoteHostIP, tarFileName, tarFileName); err != nil {
+	err = c.SSH.Copy(c.RemoteHostIP, tarFileName, tarFileName)
+	if err != nil {
 		return err
 	}
+	logger.Info("the program is sending the %s file!", tarFileName)
```

Comment:
We needn't add log here, because the Copy funcion already has log.

Reply:
OK